spec/index.html

```diff
@@ -5462,29 +5462,53 @@ <h5>RDFterm-equal</h5>
                                                                                    "type">RDF term</span> <span class="name">term1</span> <span class=
                                                                                                                                                 "operator">=</span> <span class="type">RDF term</span> <span class=
                                                                                                                                                                                                              "name">term2</span></pre>
-            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are the same RDF term as
-              defined in [[[RDF12-CONCEPTS]]] [[RDF12-CONCEPTS]]; produces a type error if the
-              arguments are both literal but are not the same RDF term <sup><a href=
-                                                                               "#func-RDFterm-equal-foot1" class="footnote">*</a></sup>; returns FALSE otherwise.
-              <code>term1</code> and <code>term2</code> are the same if any of the following is
+            <p>The behavior of this operator is defined as follows:</p>
```

Comment:
```suggestion
            <p>This function is defined as follows:</p>
```
Notice that we are not actually defining the `=` operator here but, instead, the `RDFterm-equal` function. That can be a bit confusing because the signature in the beginning of this section seems to indicate that this section is about the `=` operator. There is an open issue, #18, with a proposal to change that part of the section in order to avoid this confusion.